x-pack/plugins/security_solution_serverless/server/cloud_security/cloud_security_metering_task_config.ts

```diff
@@ -8,7 +8,7 @@
 import { cloudSecurityMetringCallback } from './cloud_security_metering';
 import type { MetringTaskProperties } from '../types';
 
-const TASK_INTERVAL = 3600; // 1 hour
+const TASK_INTERVAL = 120; // 1 hour
```

Comment:
for debugging only - will remove before merge

Reply:
@CohenIdo  Are you going to  change the `TASK_INTERVAL` back  to `3600`?

Reply:
yep:)